files/zh-cn/learn/html/howto/add_a_hit_map_on_top_of_an_image/index.md

```diff
@@ -3,6 +3,8 @@ title: Add a hit map on top of an image
 slug: Learn/HTML/Howto/Add_a_hit_map_on_top_of_an_image
 ---
 
+{{QuickLinksWithSubpages("/en-US/docs/Learn/HTML/Howto")}}
```

Comment:
This macro should use the localized path :)

See the screenshot:

![image](https://github.com/mdn/translated-content/assets/15844309/689e8db8-58f7-461b-80ba-8637c27bfdcf)

```suggestion
{{QuickLinksWithSubpages("/zh-CN/docs/Learn/HTML/Howto")}}
```